app/controllers/api/v1/skills_controller.rb

```diff
@@ -0,0 +1,9 @@
+class Api::V1::SkillsController < Api::V1::BaseController
+  respond_to :json
+
+  def index
+    featured = params[:featured]
+    respond_with featured ? Skill.where(featured: featured) : Skill.all
+  end
+
```

Comment:
Maybe for the index action we want to match the RESTful action by just sending in Skill.all. We can filter the featured parts easily with Ember.